code/menuui/barracks.cpp

```diff
@@ -459,17 +462,26 @@ void barracks_init_stats(scoring_struct *stats)
 			// Goober5000 - in case above Assert isn't triggered (such as in non-debug builds)
 			if (Num_stat_lines < Max_stat_lines)
```

Comment:
This if statement, and the Assert() above it, should be removed since they no longer matter here.